docs/Deprecating-Disabling-and-Removing-Casks.md

```diff
@@ -0,0 +1,110 @@
+
+The `because` parameter can be a preset reason (using a symbol) or a custom reason. See the [Deprecate and Disable Reasons](#deprecate-and-disable-reasons) section below for more details about the `because` parameter.
+
+## Removal
+
+A cask should be removed if it does not meet our criteria for [acceptable casks](Acceptable-Casks.md), does not install, or has been disabled for over a year.
+
+## Deprecate and Disable Reasons
+
+When a cask is deprecated or disabled, a reason explaining the action must be provided.
+
+There are two ways to indicate the reason. The preferred way is to use a pre-existing symbol to indicate the reason. The available symbols are listed below and can be found in the [`DeprecateDisable` module](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/deprecate_disable.rb):
+
+- `:does_not_build`: the cask cannot be installed on any supported macOS version.
+- `:repo_archived`: the upstream repository has been archived and no replacement is pointed to that we can use
+- `:repo_removed`: the upstream repository has been removed and no replacement is mentioned on the homepage that we can use
+- `:unmaintained`: the project appears to be abandoned i.e. it has had no commits for at least a year and has critical bugs or CVE that have been reported and gone resolved longer. Note: some software is "done"; a lack of activity does not imply a need for removal.
+- `:unsupported`: Homebrew's compilation of the software is not supported by the upstream developers (e.g. upstream only supports macOS versions older than 10.15)
+- `:deprecated_upstream`: the project is deprecated upstream and no replacement is pointed to that we can use
+- `:checksum_mismatch`: the checksum of the source for the casks's current version has changed since bottles were built and we cannot find a reputable source or statement justifying this
+
+These reasons can be specified by their symbols (the comments show the message that will be displayed to users):
+
+```ruby
+# Warning: <cask> has been deprecated because it is deprecated upstream!
+deprecate! date: "2020-01-01", because: :deprecated_upstream
+```
+
+```ruby
+# Error: <cask> has been disabled because it does not build!
+disable! date: "2020-01-01", because: :deprecated_upstream
+```
+
+If these pre-existing reasons do not fit, a custom reason can be specified. Such reasons should be written to fit into the sentence `<cask> has been deprecated/disabled because it <reason>!`.
+
+A well-worded example of a custom reason would be:
+
+```ruby
+# Warning: <cask> has been deprecated because it fetches unversioned dependencies at runtime!
+deprecate! date: "2020-01-01", because: "fetches unversioned dependencies at runtime"
+```
+
+A poorly-worded example of a custom reason would be:
+
+```ruby
+# Error: <cask> has been disabled because it invalid license!
+disable! date: "2020-01-01", because: "invalid license"
```

Comment:
Could be a few more here:

```ruby
:licence_invalid
:license_unknown 
:license_incompatible
```

Reply:
We don't track licenses for casks

Reply:
👍 This suggestion could also apply to the formula section below.